policies/cla.html

```diff
@@ -34,9 +34,19 @@
           </p>
 
           <p>
-          A submission is considered trivial if the author and all reviewers
-          agree that it is. In this case a CLA is not required and the
-          commit text should include "CLA: trivial" on a line by itself.
+	  A submission is trivial if it is considered trivial under copyright
```

Comment:
Indentation or whitespace issue. (Applies to entire patch)

Reply:
It's whitespace, the file is using tabs and spaces.  I used spaces for some of it.